_posts/2020-02-22-gsoc2020.md

```diff
@@ -0,0 +1,113 @@
+and with different skill requirements. However, check the [ideas][]
+page — we might add more ideas after the posting date of this post.
+
+You don't need to have all the skills that we are listing, although
+that helps, of course. But you need to demonstrate to us that you're
+able and keen to learn anything that you don't know yet, and we will
+be happy to help you learn during your project with us.
+
+| project | name | difficulty | description | skills | mentors |
+|---------|---------------------------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|---------------------------|
+| 1 | [Molecular volume and surface analysis]({{ site.github.wiki }}/Project-Ideas-2020#project-1-molecular-volume-and-surface-analysis) | easy | use an existing package for molecular surface area calculations to build a new analysis module | Python, MDAnalysis.analysis | @orbeckst @IAlibay @richardjgowers
+| 2 | [Command Line Interface (CLI)]({{ site.github.wiki }}/Project-Ideas-2020#project-2-command-line-interface-cli) | easy | create a new package with MDAnalysis-based command line tools | Python, bash, Python packages, CI, MD | @PicoCentauri @orbeckst @fiona-naughton
+| 3 | [Interoperability with RDKit]({{ site.github.wiki }}/Project-Ideas-2020#project-3-interoperability-with-rdkit) | medium | add capability to MDAnalysis to use the RDKit API to convert data structures between MDAnalysis and RDKit | MDAnalysis, RDKit, Python, C++ (?) | @richardjgowers @IAlibay
+| 4 | [Improved atom selections]({{ site.github.wiki }}/Project-Ideas-2020#project-4-improved-atom-selections) | medium | replace the selection parsing code with a more flexible parser | Python | @orbeckst, @IAlibay, @fiona-naughton
+| 5 | [Serialize Universes for parallel computation]({{ site.github.wiki }}/Project-Ideas-2020#project-5-serialize-universes-for-parallel-computation) | medium/challenging | make the central `Universe` data structure serializable with Pickle to enable simple parallelization with Dask or MPI | Python, MDAnalysis I/O, task-based parallelization | @richardjgowers, @orbeckst, @IAlibay, @fiona-naughton 
+| 6 | [Implement TNG support]({{ site.github.wiki }}/Project-Ideas-2020#project-6-implement-tng-support) | challenging | write Cython bindings to the TNG library and write classes to bring this information into MDAnalysis | Cython, C++ | @richardjgowers
+
```

Comment:
This requires C knowledge not C++. However the tng library is not the easiest to understand. Lots of manual memory management. This also caused our delays as I do not know where the segfault in pytng comes from.

Reply:
This project also requires knowledge about md simulation or at least the willingness to learn a lot about it in a very short time.